src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -9,7 +9,9 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final int MAX_CHARACTERS = 45;
+    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric and be within " + MAX_CHARACTERS
```

Comment:
Maybe rephrase to "maximum of 45 characters" instead of "be within 45 characters"?

Reply:
done